approval_polls/static/approval_polls/create.js

```diff
@@ -120,7 +126,10 @@ $(function () {
     console.log(e);
     var container = $(e.currentTarget).closest('.input-group');
     container.remove();
-    that.numChoiceFields--;
```

Comment:
I believe @Faxn is working to iron out this issue of choice numbers, so would you like to introduce your changes after his are committed ?